### PR TITLE
No force unwrapping

### PR DESCRIPTION
**Pull request by ntnmrndn**

### CODE REVIEW

File: Capturer/Basic/Inputs/CameraInput.swift [hunk 1]
```diff
@@ -30,25 +30,29 @@ extension CameraInput {
...
-        case .back: return .back
+        case .front: .front
+        case .back: .back
         }
-      }()
-    )
-
-    let device = discoverySession.devices.first!
-
+      }())
```
Comment (JohnEstropia) [inline Capturer/Basic/Inputs/CameraInput.swift:51]: You won't need the anonymous closure anymore either 👍

**Review by JohnEstropia — APPROVED** [review]

File: Capturer/Basic/Inputs/CameraInput.swift (hunk 1 above)
Comment (ntnmrndn) [reply Capturer/Basic/Inputs/CameraInput.swift:51]: @JohnEstropia actually we do, if I try 

```
    let discoverySession = AVCaptureDevice.DiscoverySession(
      deviceTypes:
        [
          .builtInWideAngleCamera,
          .builtInUltraWideCamera,
          .builtInTelephotoCamera
        ],
      mediaType: .video,
      position: switch position {
        case .front: .front
        case .back: .back
        }
   )
```
I get 
> 'switch' may only be used as expression in return, throw, or as the source of an assignment

Comment (JohnEstropia) [reply Capturer/Basic/Inputs/CameraInput.swift:51]: Huh, I see. I guess there's still an unreleased Swift Evolution proposal

Comment (JohnEstropia) [reply Capturer/Basic/Inputs/CameraInput.swift:51]: Aww, it's in [Future Directions](https://github.com/apple/swift-evolution/blob/main/proposals/0380-if-switch-expressions.md#full-expressions)

**Pull request merged**